examples/basic/src/main/java/io/littlehorse/examples/BasicExample.java

```diff
@@ -64,36 +65,38 @@ public static void main(String[] args) throws IOException {
 
         // New worker
         LHTaskWorker worker = getTaskWorker(config);
-
-        // Register task if it does not exist
-        if (worker.doesTaskDefExist()) {
-            log.debug(
-                "Task {} already exists, skipping creation",
-                worker.getTaskDefName()
-            );
-        } else {
-            log.debug(
-                "Task {} does not exist, registering it",
-                worker.getTaskDefName()
-            );
-            worker.registerTaskDef();
-        }
+//
```

Comment:
Ooops This is not supposed to be commented

Reply:
We're going to make `PutTaskDef` idempotent too (:

Reply:
:tada: